backend/app/views/spree/admin/stores/_form.html.erb

```diff
@@ -9,7 +9,7 @@
   <div class="card-body">
     <%= f.field_container :logo, class: ['form-group'] do %>
       <% if @store.logo.attached? %>
-        <%= image_tag main_app.url_for(@store.logo.variant(resize: '244x104>')) %>
```

Comment:
@MatthewKennedy hmm this resize here was a safeguard, if someone would upload a huge file

Reply:
With an svg the resize will cause an error, But you can validate file size with eh active_storage_validations gem?